### PR TITLE
Added Grid Cell Component

### PR DESCRIPTION
**Pull request by fran-6co**
[https://github.com/users/watchorpass/projects/1?pane=issue&itemId=74319162](<#URL#> "Task link")

### CODE REVIEW

**Comment by fran-6co** [comment]
Improved commit for review, @Bitos88 @franmu94 @alexmaxu

File: MovieFlick/GridCellComponent.swift [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    )
+    
+    var body: some View {
+        ZStack(alignment: .bottomLeading) {
+            (image ?? Image(systemName: "popcorn"))
+                .resizable()
+                .scaledToFit()
+                .padding()
+                .frame(width: cellSize, height: cellSize)
+                .background(.ultraThinMaterial) // por si sale el placeholder popcorn, que se vea el fondo difuminado.
```
[the remark this answers is not in the document]
Comment (fran-6co) [reply MovieFlick/GridCellComponent.swift]: No hay comentarios en el último commit.

File: MovieFlick/GridCellComponent.swift [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            }
+            Text(title)
+                .lineLimit(2)
+                .font(.title)
+                .fontWeight(.semibold)
+                .padding(.vertical, 10)
+                .allowsTightening(true)
+                .truncationMode(.middle)
+                .frame(maxWidth: .infinity)
+                .background(.thinMaterial)
```
Comment (Bitos88) [inline MovieFlick/GridCellComponent.swift:35]: Ui esto...

**Review by Bitos88 — CHANGES_REQUESTED** [review]

File: MovieFlick/GridCellComponent.swift [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    var action: () -> Void
+    
+    var gradient = LinearGradient(
+        colors: [.black.opacity(0.8), .black.opacity(0)],
+        startPoint: .bottom,
+        endPoint: .center
+    )
+    
+    var body: some View {
+        ZStack(alignment: .bottomLeading) {
```
[the remark this answers is not in the document]
Comment (fran-6co) [reply MovieFlick/GridCellComponent.swift]: commit antiguo

**Comment by Bitos88** [comment]
image

**Review by alexmaxu — APPROVED** [review]

**Review by Bitos88 — APPROVED** [review]

**Pull request merged**